src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -16,11 +16,17 @@
 
 import java.math.BigDecimal;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Comparator;
 import java.util.List;
+import java.util.Map;
 import java.util.PriorityQueue;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import net.htmlparser.jericho.*;
```

Comment:
import only the libraries you need: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports